operating/manage-vulnerabilities/common-vuln-management-tasks.adoc

```diff
@@ -34,14 +34,14 @@ include::modules/vulnerability-management20-view-platform-cve.adoc[leveloffset=+
 
 You can exclude or ignore CVEs in {product-title-short} by snoozing node and platform CVEs and deferring or marking node, platform, and image CVEs as false positives. You might want to exclude CVEs if you know that the CVE is a false positive or you have already taken steps to mitigate the CVE. Snoozed CVEs do not appear in vulnerability reports or trigger policy violations.
 
-You can snooze a CVE to globally ignore it for a specified period of time. Snoozing a CVE does not require approval.
+You can snooze a CVE to ignore it globally for a specified period of time. Snoozing a CVE does not require approval.
 
 [NOTE]
 ====
-Snoozing node and platform CVEs requires enabling the ROX_VULN_MGMT_LEGACY_SNOOZE environment variable.
+Snoozing node and platform CVEs requires that the ROX_VULN_MGMT_LEGACY_SNOOZE environment variable is enabled.
```

Comment:
Backticks for ROX_VULN_MGMT_LEGACY_SNOOZE, since it's a variable?

Reply:
Yes, good catch, thanks!